src/Transform.h

```diff
@@ -82,6 +84,7 @@ class Transform : public std::enable_shared_from_this<Transform>
 
     void set_dirty();
     void set_parent_dirty();
+    void set_children_dirty();
```

Comment:
We can add this method in a separate commit and replace other manual ways of doing this.